<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>291</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="listenTab">
     <attribute name="title">
      <string>Listen</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="waterfallTab">
     <attribute name="title">
      <string>Waterfall</string>
     </attribute>
     <widget class="QSlider" name="bandwidthSlider">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>230</y>
        <width>450</width>
        <height>16</height>
       </rect>
      </property>
      <property name="maximum">
       <number>450</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="frequencySlider">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>160</y>
        <width>450</width>
        <height>16</height>
       </rect>
      </property>
      <property name="maximum">
       <number>450</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLCDNumber" name="centerFreqLcdNumber">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>130</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="digitCount">
       <number>6</number>
      </property>
     </widget>
     <widget class="QLCDNumber" name="bandwidthLcdNumber">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>200</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="digitCount">
       <number>6</number>
      </property>
     </widget>
     <widget class="QLabel" name="centerFreqLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>110</width>
        <height>19</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Center Freq.</string>
      </property>
     </widget>
     <widget class="QLabel" name="bandwidthLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>110</width>
        <height>19</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Bandwidth</string>
      </property>
     </widget>
     <widget class="QLabel" name="waterfallLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>450</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string>No Spectal Data</string>
      </property>
     </widget>
     <widget class="QLabel" name="waterfallFreqLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>450</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0Hz</string>
      </property>
     </widget>
     <widget class="QLabel" name="centerFreqUnitsLabel">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>140</y>
        <width>41</width>
        <height>19</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>MHz</string>
      </property>
     </widget>
     <widget class="QLabel" name="bandwidthUnitsLabel">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>210</y>
        <width>41</width>
        <height>19</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>kHz</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
